spec/lib/unit_tests_utils/consul_spec.rb

```diff
@@ -3,16 +3,18 @@
 
 describe UnitTestsUtils::Consul do
   let(:internal_consul_ip) { "1.2.3.4" }
-
-  before :each do
```

Comment:
If i got you right, you want that we make the before each and after each hooks consistent. I commited brackets into every before and after hook here https://github.com/anynines/unit-tests-utils/pull/3/commits/957a42c46c806bef473039e13e6e985d773e36fe .